website/content/en/docs/about/under-the-hood/guarantees.md

```diff
@@ -11,6 +11,33 @@ Here you can find an overview of delivery guarantee types and their meaning as w
 the stability of our components. Next, you can head over to the [components] page and use filters to
 see which components support specific guarantees.
 
+## Acknowledgement guarantees
+
+Vector supports end-to-end acknowledgement for the majority of its
+sources and sinks. This is a system which tracks the delivery status of
+an event through the lifetime of that event as it travels from the
+originating source to any number of destination sinks. Support for this
+feature is indicated by the "acknowledgements" badge at the top of the
+relevant sink or source configuration reference page.
+
+For a source that supports end-to-end acknowledgements and has the
+`acknowledgements` option enabled, this will cause it to wait for _all_
```

Comment:
Is this true? I thought we moved the `acknowledgements` option to the sinks?

Reply:
It is still true, but incomplete and deprecated. I will reword this.